heartbeat/Jenkinsfile.yml

```diff
@@ -37,6 +37,7 @@ stages:
         stage: extended
     unitTest:
         mage: "mage build unitTest"
+        withNodejs: true
```

Comment:
Technically this doesn't make a difference for non-xpack heartbeat since that doesn't support synthetic monitors, but it certainly doesn't hurt.

Reply:
I'll then remove it

Reply:
https://github.com/elastic/beats/pull/32710/commits/94005c0d5aaa6196d0d1832305ee035642bb4adc